internal/uvm/create.go

```diff
@@ -35,9 +36,10 @@ func (uvm *UtilityVM) OS() string {
 // Close terminates and releases resources associated with the utility VM.
 func (uvm *UtilityVM) Close() error {
 	uvm.Terminate()
-	if uvm.gcslog != nil {
-		uvm.gcslog.Close()
-		uvm.gcslog = nil
+	if uvm.outputListener != nil {
```

Comment:
I don't understand how this will ever be true. In `Start()` you set it to `nil` after forwarding it to the process call.

Reply:
I think it's there in case you do Create -> Stop without a Start in between.

Reply:
Can you add a comment in Stop:
```go
// outputListener will only be nil for a Create -> Stop without a Start. In this case
// we have no goroutine processing output so its safe to close the channel here.
if uvm.outputListener != nil {
  // ...
}
```